R/mod.prevalence.R

```diff
@@ -156,6 +156,10 @@ prevalence_msm <- function(dat, at) {
   dat$epi$ir100.uct[at] <- (dat$epi$incid.uct[at] / sum(uCT == 0, na.rm = TRUE)) * 5200
   dat$epi$ir100.ct[at] <- (dat$epi$incid.ct[at]/ sum(rCT == 0 | uCT == 0, na.rm = TRUE)) * 5200
 
+  dat$epi$prev.sti[at] <- sum(rGC == 1 | ugc == 1 | rCT ==1 | uCT == 1, na.rm = TRUE) / dat$epi$num[at]
+  dat$epi$ir100.sti[at] <- ((dat$epi$incid.ct[at] + dat$epi$incid.gc)/ sum(rCT == 0 | uCT == 0 | rGC == 0 | uGC ==0, na.rm = TRUE)) * 5200
```

Comment:
Actually, the denominator for the incidence stats should link the STIs with an `&` not an `|` (my mistake). Can you change for the other incidence stats too?